src/tools/illink/test/Mono.Linker.Tests/TestCasesRunner/ResultChecker.cs

```diff
@@ -880,7 +880,7 @@ void VerifyLoggedMessages (AssemblyDefinition original, TrimmingTestLogger logge
 											break;
 										}
 										if (memberDefinition.Name == ".ctor" &&
```

Comment:
Nit: This should probably also have `actualName.StartsWith (expectedMember.DeclaringType.FullName)` just like the static `.cctor` above.